clerk/tokens.go

```diff
@@ -2,50 +2,52 @@ package clerk
 
 import (
 	"fmt"
+	"strings"
 	"time"
 
 	"gopkg.in/square/go-jose.v2"
 
 	"gopkg.in/square/go-jose.v2/jwt"
 )
 
-type Claims struct {
+var (
+	standardClaimsKeys = []string{"iss", "sub", "aud", "exp", "nbf", "iat", "jti"}
+)
+
+type TokenClaims struct {
 	jwt.Claims
-	UserClaims
+	Extra map[string]interface{}
```

Comment:
🔧 Instead of using an extra field, can't we just have this be a `map`? Since this is for decoding any JWT really..

Reply:
I decided to embed the standard claims, just to make users life easier so they don't need to check if exists in the map

Reply:
Fair point!